ENCM-27 Show only released results for cards and home-page searches

forresttanaka opened the pull request:
Change Log

* Change “Experiment search” under the “Functional Characterization data” menu to show onlly released datasets.
* Adjust queue_worker_processes from 24 to 20 to avoid OOM errors on demo deploy.
* Add status=released to any home-page card searches that didn’t already have that.
* Update BDD tests for new home-page searches.

forresttanaka committed:
ENCM-27-fcc-released

Change Log

* Change “Experiment search” under the “Functional Characterization data” menu to show onlly released datasets.
* Adjust queue_worker_processes from 24 to 20 to avoid OOM errors on demo deploy.
* Add status=released to any home-page card searches that didn’t already have that.
* Update BDD tests for new home-page searches.

forresttanaka committed:
ENCM-27-fcc-released

Added more “status=released” to the menu items that don’t have them.

forresttanaka commented:
@mingjiecn Code review update: Meenakshi found some links that still need “status=released” added, so I’ve added those so that all search-based menu items have status=released, except for RNA Get.

mingjiecn commented on src/encoded/static/components/app.js:
```diff
@@ -58,13 +58,13 @@ const portal = {
-                { id: 'mouse-development-matrix', title: 'Mouse development matrix', url: '/mouse-development-matrix/?type=Experiment&status=released&related_series.@type=OrganismDevelopmentSeries&replicates.library.biosample.organism.scientific_name=Mus+musculus', tag: 'collection' },
+                { id: 'reference-epigenomes-human', title: 'Human reference epigenomes', url: '/reference-epigenome-matrix/?type=Experiment&control_type!=*&related_series.@type=ReferenceEpigenome&replicates.library.biosample.donor.organism.scientific_name=Homo+sapiens&status=released', tag: 'collection' },
+                { id: 'reference-epigenomes-mouse', title: 'Mouse reference epigenomes', url: '/reference-epigenome-matrix/?type=Experiment&control_type!=*&related_series.@type=ReferenceEpigenome&replicates.library.biosample.donor.organism.scientific_name=Mus+musculus&status=released', tag: 'collection' },
+                { id: 'mouse-development-matrix', title: 'Mouse development matrix', url: '/mouse-development-matrix/?type=Experiment&status=released&related_series.@type=OrganismDevelopmentSeries&replicates.library.biosample.organism.scientific_name=Mus+musculus&status=released', tag: 'collection' },
```
duplicated "status=released" in Mouse development matrix url

mingjiecn reviewed (changes requested):
duplicated "status=released" in Mouse development matrix url

forresttanaka committed:
ENCM-27-fcc-released

Removed the newly duplicated status=released from the mouse development matrix link.

forresttanaka commented:
@mingjiecn Thanks for finding that duplication!
